app/workers/tipline_statistics_periodic_check_worker.rb

```diff
@@ -0,0 +1,13 @@
+class TiplineStatisticsPeriodicCheckWorker
+  include Sidekiq::Worker
+
+  sidekiq_options retry: 0
+
+  def perform
+    if CheckConfig.get('ENABLE_TIPLINE_STATS_MONITORING', true)
```

Comment:
Is this needed? This job won't be scheduled by default, it will execute only if it's explicitly created.

Reply:
Removed